data/sql/updates/pending_db_world/rev_1660996114567482800.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `gameobject` WHERE `id`=180745;
```

Comment:
delete GUID and ID as GUID is pri

Reply:
And what's the difference here?

Reply:
yea but you do not delete the GUID, which can cause SQL errors as GUID is the Primary Key =)